### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by tcrasset** [Ezamey/Wine_Testing#1]
# Structured data review

You did a fantastic job, that's why I allowed myself to be a bit more finnicky with the things to improve. Hope you see it as a way to be EVEN better

## Good

- Really like the directory architecture
- Good job for `requirements.txt`
- EXCELLENT README, installation, analysis performed, etc...
- Nice little touch of adding a new feature
- Good idea checking other metrics than accuracy
- Nice plots
- Good use of validation set
- Nice try with random sampling, but it doesn't seem to compile
- VERY NICE, you used cross validation
- I like that you created a separate `.py` file for the model

## To improve

- I would have separated EDA into it's own file
- In the README, a graph of the accuracies/metrics would have been nice, comparing each modification/approach/baseline rather than showing the print result of the keras training output
- You created `modelkeras.py` but you never use it inside notebooks
- You could have reduced code duplication even more by creating functions, such as `get_data()` where you do all the preprocessing, or others. These could have been stored in `.py` files and called in different notebooks
